.github/workflows/regression-report.yml

```diff
@@ -225,8 +229,8 @@ jobs:
   # should be always last step
   send_message:
     runs-on: ubuntu-latest
-    needs: [ parse_comment, make_report ]
-    if: failure() || success()
+    needs: [ check_pr_status, parse_comment, make_report ]
+    if: always()
     steps:
```

Comment:
Let's make separate job to send sad emoji ` if canceled()`
Let's not mix it within other logic.

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/expressions#example-of-cancelled

Reply:
done